bedrock/contentful/templates/includes/contentful/cta.html

```diff
@@ -32,5 +31,10 @@
 
   <a href="{{ url('products.vpn.landing') }}" class="mzp-c-button {{ button_class }}" data-cta-text="{{ cta_text }}" data-cta-type="link" data-cta-position="{{ location }}">{{ label }}</a>
 
+{% elif action == 'Try Relay' %}
+
+  {% set referral = '?utm_source=' + utm_source + '&utm_medium=referral&utm_campaign=' + utm_campaign %}
```

Comment:
Rather than repeat this here (as well as on line 27), maybe we could use one common variable that each CTA type can use to set referral params?